tests/integration_tests/test_retrieve_submission_file.py

```diff
@@ -1,11 +1,26 @@
 import io
 
 import pytest
+from botocore.exceptions import ClientError
 
 from config import Config
 from core.aws import _S3_CLIENT
 from core.const import EXCEL_MIMETYPE
 from core.db.entities import Submission
+from tests.integration_tests.conftest import create_bucket, delete_bucket
+
+
+@pytest.fixture(autouse=True, scope="module")
+def test_buckets():
+    """Sets up and tears down buckets used by this module.
+    On set up:
+    - creates data-store-successful-files-unit-tests
+    On tear down, deletes all objects stored in the buckets and then the buckets themselves.
+    """
+
+    create_bucket(Config.AWS_S3_BUCKET_SUCCESSFUL_FILES)
+    yield
+    delete_bucket(Config.AWS_S3_BUCKET_SUCCESSFUL_FILES)
```

Comment:
best to use `test_buckets` defined in `conftest.py` (this was added in a recent commit by @MarcUsher)

Reply:
Good spot @cyrusdobbs looks like this fixture was no longer needed, so I've removed and force-pushed.